Irrgarten_ruby/Player.rb

```diff
@@ -0,0 +1,115 @@
+module Irrgarten
+class Player
+    @@MAX_WEAPONS = 2
+    @@MAX_SHIELDS = 3
+    @@INITIAL_HEALTH = 10
+    @@HITS2LOSE = 3
+    def initialize(number, intelligence, strength, health, row, col) 
+        @number = number
+        @name = "Player #{@number}"
+        @intelligence = intelligence
+        @strength = strength
+        @health = health
+        @row = row
+        @col = col
+        @consecutive_hits = 0
+        @weapons = Array.new
+        @shields = Array.new
```

Comment:
```suggestion
        self.resurrect
        set_pos(0,0)
```
Cambios para reutilizar código. La gran pregunta de si la posición debe ser inicializada ya la solucionaremos.